verification/curator-service/ui/src/components/AutomatedBackfill.tsx

```diff
@@ -148,10 +161,36 @@ export default function AutomatedBackfill(props: Props): JSX.Element {
                 setSuccessMessage(
                     baseMessage.concat(createMessage, updateMessage),
                 );
+            } else if (uploadStart && outOfTime(uploadStart)) {
+                setUploadStatus('ERROR');
+                setErrorMessage(
+                    `Upload ${uploadId} failed with timeout.
+                     This typically means that the parsing function wasn't able to ingest all cases in the provided range within the function time limit.
+                     Please search the line list by the above upload ID to find what dates were covered, and for subsequent backfills, try running over a date range with fewer cases.`,
```

Comment:
we should probably sort cases searched by uploadids by confirmed date desc to make it human-searchable, not in this PR but surely a nice later addition.

Reply:
Yeah, that'd certainly be useful. Filed #1297